src/librustdoc/src/lib.rs

```diff
@@ -741,30 +740,35 @@ where R: 'static + Send,
             }
         }
 
-        // Load all plugins/passes into a PluginManager
-        let path = plugin_path.unwrap_or("/tmp/rustdoc/plugins".to_string());
-        let mut pm = plugins::PluginManager::new(PathBuf::from(path));
+        if !plugins.is_empty() {
+            eprintln!("WARNING: --plugins no longer functions; see CVE-2018-1000622");
+        }
+
+        if !plugin_path.is_none() {
+            eprintln!("WARNING: --plugin-path no longer functions; see CVE-2018-1000622");
+        }
+
+        info!("Executing passes");
```

Comment:
Why this add?

Reply:
It wasn't added; it was in the old code. It used to say passes/plugins

Reply:
Oh sorry! Damn diff...